src/components/SortBy/SortBy.jsx

```diff
@@ -1,20 +1,26 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import classNames from 'classnames/bind';
+import PropTypes from 'prop-types';
 import styles from './SortBy.scss';
+import * as constants from '../../constants.json';
+import sort from '../../actions/sort';
 
 const cx = classNames.bind(styles);
 
 class SortBy extends Component {
   constructor(props) {
     super(props);
     this.state = {
```

Comment:
We don't need local state here, because we save state of this component to the redux store